diff --git a/mmyolo/__init__.py b/mmyolo/__init__.py
index 6a0bd5d..c15d4a2 100644
--- a/mmyolo/__init__.py
+++ b/mmyolo/__init__.py
@@ -7,7 +7,7 @@ from mmengine.utils import digit_version
 from .version import __version__, version_info
 
 mmcv_minimum_version = '2.0.0rc4'
-mmcv_maximum_version = '2.1.0'
+mmcv_maximum_version = '2.2.0'
 mmcv_version = digit_version(mmcv.__version__)
 
 mmengine_minimum_version = '0.7.1'
@@ -20,7 +20,7 @@ mmdet_version = digit_version(mmdet.__version__)
 
 
 assert (mmcv_version >= digit_version(mmcv_minimum_version)
-        and mmcv_version < digit_version(mmcv_maximum_version)), \
+        and mmcv_version <= digit_version(mmcv_maximum_version)), \
     f'MMCV=={mmcv.__version__} is used but incompatible. ' \
     f'Please install mmcv>={mmcv_minimum_version}, <{mmcv_maximum_version}.'
 
